python_modules/dagster/dagster/core/execution.py

```diff
@@ -707,6 +707,13 @@ def execute_plan(execution_plan, environment_dict=None, run_config=None, step_ke
     run_config = check_run_config_param(run_config)
     check.opt_list_param(step_keys_to_execute, 'step_keys_to_execute', of_type=str)
 
+    if step_keys_to_execute:
+        for step_key in step_keys_to_execute:
+            if not execution_plan.has_step(step_key):
```

Comment:
👍 we *could* collect all the errors & say "execution plan does not contain steps..."

Reply:
agree. let's do that in its own change. https://github.com/dagster-io/dagster/issues/987